CHANGELOG.md

```diff
@@ -42,6 +44,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `test_utils::contracts` now creates a `Move.lock` if it doesn't exist yet
 - Fixed a bug that erases the current basic auth credentials from the config when any value is updated
 
+### `nexus-toolkit-rust`
+
+#### Added
+
+- `/tools` endpoint to the `boostrap!` macro that returns a list of all tools registered on the webserver
```

Comment:
I need to insist, for the sake of making everything extra clear for the consumer of the SDK. 

By this PR, `/tools` is relevant only if someone is using the Rust toolkit. But if I am implementing a Tool in Python, then I do not know about this endpoint and in such a case the `--batch` option is rendered erroneous (it will fail).

How can we introduce `/tools` in a clearer way? 

One answer to the above could be: mention `/tools` and it's relation to the Rust toolkit in the description of the `--batch` option, and call it a day. 

It can be as simple as that but there has to be something, somewhere ...

cc: @tmsdkeys

Reply:
OK, please disregard. I missed this https://github.com/Talus-Network/nexus-sdk/commit/438a55b58228cc6f0da47f024407bcea07477cc3

All good for now.